Add hostname MOTD banner

pfeerick opened the pull request:
Adding the hostname MOTD banner from the rock64

ayufan commented:
Please don't depend on `figlet`. It will fail when installing `.deb` package manually. Instead, add `figlet` to base system dependencies (in make_rootfs.sh) and add a condition which checks if figlet is installed and shows prompty only when is.

pfeerick commented:
Hm... we'll have to remove that on the rock64 then, as that is how you did it over there ;) It's already there in the base system dependencies, that was the only thing that had been missed when I added it (back when I added htop).

ayufan commented:
Rock64 uses different update mechanism, using apt-get so adding
dependencies is safe then.

On Wed, 26 Jul 2017 at 13:47, Peter <notifications@github.com> wrote:

> Hm... we'll have to remove that on the rock64 then, as that is how you did
> it over there ;) It's already there in the base system dependencies, that
> was the only thing that had been missed when I added it (back when I added
> htop).
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/ayufan-pine64/linux-build/pull/33#issuecomment-318029826>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACTpQe3h73uV0TzAkI_Rv1nNMgVEQvujks5sRydBgaJpZM4OjyfJ>
> .
>

ayufan commented on package/root/etc/update-motd.d/05-figlet:
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+if [ $(which figlet) ]; then
```
`if which figlet 1>/dev/null 2>/dev/null; then`

We don't want to show figlet in motd.

pfeerick replied:
We don't? :-P

ayufan replied:
execution of `which figlet` which generates: :)

```
$ which figlet
figlet not found
$ which bash
/usr/local/bin/bash
```

pfeerick replied:
I did commit the change incase you didn't notice ;)

ayufan replied:
Still, I will be nitpicky, why `[ $(`? :)

pfeerick replied:
Oh raspberry! It's your repo... be picky all you want :-P
